src/iidm/converter/xml/StaticVarCompensatorXml.cpp

```diff
@@ -63,6 +76,13 @@ void StaticVarCompensatorXml::writeRootElementAttributes(const StaticVarCompensa
     writePQ(svc.getTerminal(), context.getWriter());
 }
 
+void StaticVarCompensatorXml::writeSubElements(const StaticVarCompensator& svc, const VoltageLevel& /*voltageLevel*/, NetworkXmlWriterContext& context) const {
+    IidmXmlUtil::assertMinimumVersionAndRunIfNotDefault(!stdcxx::areSame(svc, svc.getRegulatingTerminal().getConnectable().get()),
+        STATIC_VAR_COMPENSATOR, REGULATING_TERMINAL, xml::ErrorMessage::NOT_DEFAULT_NOT_SUPPORTED, IidmXmlVersion::V1_1(), context.getVersion(), [&svc, &context]() {
```

Comment:
Is it really necessary to prefix ErrorMessage using xml?

Reply:
No it's not